apps/blog/_posts/2026/01/coding-agent/long-running-agent.md

```
@@ -0,0 +1,178 @@
+[Continuous Claude](https://github.com/AnandChowdhary/continuous-claude), 
+[Continuous-Claude-v3](https://github.com/parcadei/Continuous-Claude-v3)
+
+Good reads:
+
+- [Scaling long-running autonomous coding](https://cursor.com/blog/scaling-agents)
+- [Anthropic CEO on Claude, AGI & the Future of AI & Humanity | Lex Fridman Podcast #452](https://lexfridman.com/dario-amodei-transcript)
+- [2025 LLM Year in Review - Andrej Karpathy](https://karpathy.bearblog.dev/year-in-review-2025/)
+
+Next: [Claws](/2026/02/claws.html)
+
+### Claude Code + Ralph Loop
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)